InteractionBase/src/commands/CNewView.cpp

```diff
@@ -38,9 +39,9 @@ CommandResult* CNewView::executeWithArguments(Visualization::Item *, Visualizati
 		const QStringList& arguments, const std::unique_ptr<Visualization::Cursor>&)
 {
 	bool open = arguments.at(1).compare("open", Qt::CaseInsensitive) == 0;
-	auto view = target->scene()->newViewItem(arguments.at(0));
+	auto view = target->scene()->viewItemManager()->newViewItem(arguments.at(0));
```

Comment:
The name of the class `ViewItemManager` is OK. However I think the accessor method in `Scene` should have a shorter name, since it will be somewhat often used. It would have been ideal to call it `views` but that method already exists in `QGraphicsScene` which `Scene` inherits. Do you have a good idea for a shorter name? What about:
- `viewGrid`
- `viewItems`
- `viewManager` (not that short)
- ...